src/test/java/uk/gov/hmcts/sptribs/notification/NotificationServiceCICTest.java

```diff
@@ -255,50 +253,94 @@ void shouldThrowNotificationExceptionWhenClientFailsToSendEmail()
             any());
     }
 
-    @Ignore
-    void shouldThrowNotificationExceptionWhileFileUploadToSendEmail()
-        throws IOException {
+    @Test
+    void shouldThrowIOExceptionWhenClientFailsToSendEmail()
```

Comment:
Please also add a test that covers the condition when an IOException is thrown

Reply:
As per Gabrielle's comment this is not testing IOException - it is testing NotificationException. Please change the name and add a test case for IOException.

Reply:
Apologies, I changed the name of the method but it seems that the implementation of the method sendEmail throws the IOException wrapped in a notificationException. So I haven't been able to find a way to just test IOException without the notificationException being present as well, maybe I am missing something.